taskRequests/details/script.js

```diff
@@ -111,18 +127,18 @@ async function renderTaskDetails(taskId) {
 }
 
 function getAvatar(user) {
-  if (user.user?.picture?.url) {
+  if (user?.user?.picture?.url) {
     return createCustomElement({
       tagName: 'img',
-      src: user.user.picture.url,
-      alt: user.user.first_name,
-      title: user.user.first_name,
+      src: user?.user?.picture?.url,
+      alt: user?.user?.first_name,
+      title: user?.user?.first_name,
```

Comment:
- can we use the `??` operator here?

Reply:
Can you please provide an example? 

From what I understood if we do something like this:
```javascript
 user.user.picture.url ?? ''
``` 
won't there be an error if `picture` was `undefined` and we are try to access `url` from `undefined`?